changelog.txt

```diff
@@ -12,6 +12,7 @@
 * Update - Implement expirement on Connect Page.
 * Fix - Columns are misaligned on Payments->Transactions/Disputes page.
 * Fix - Risk level is displayed as a "Numeric" value in transactions CSV.
+* Fix - Extra Column without a title is displayed in downloaded CSV file for the transaction table (Added a column with Transaction Id for CSV file).
```

Comment:
Let's make the message a bit more succint, e.g. `Fix - Add CSV column heading for transaction id column.`